### PR TITLE
fix: add Goutam Nath

### PR DESCRIPTION
**Pull request by nathgoutam93**
## Changes proposed

added new profile Goutam Nath

## Check List

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/809"></a>

### CODE REVIEW

File: public/data/nathgoutam93.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ],
+    "milestones": [
+      {
+        "title":"Started Game Developing",
+        "date": "August 2020",
+        "color": "orange",
+        "description": "learned Unreal engin 4 and blender 3D"
+      },
+      {
+        "title": "Started Web developing",
```
Comment (Pradumnasaraf) [inline public/data/nathgoutam93.json:31]:
```suggestion
        "title": "Started Web Development",
```

File: public/data/nathgoutam93.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "icon": "globe",
+        "color": "blue",
+        "description": "got to learn about Web dev tools like Html, css, js, node and now playing with js/css frameworks"
+      },
+      {
+        "title": "Started Open Source",
+        "date": "December 2021",
+        "icon": "github",
+        "color": "green",
+        "description": "Started Contibuting To open source projects"
```
Comment (Pradumnasaraf) [inline public/data/nathgoutam93.json:42]:
```suggestion
        "description": "Started contributing to Open Source projects"
```

File: public/data/nathgoutam93.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      {
+        "title": "Started Web developing",
+        "date": "october 2021",
+        "icon": "globe",
+        "color": "blue",
+        "description": "got to learn about Web dev tools like Html, css, js, node and now playing with js/css frameworks"
+      },
+      {
+        "title": "Started Open Source",
```
Comment (Pradumnasaraf) [inline public/data/nathgoutam93.json:38]:
```suggestion
        "title": "Started Open Source contribution",
```

File: public/data/nathgoutam93.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        "date": "August 2020",
+        "color": "orange",
+        "description": "learned Unreal engin 4 and blender 3D"
+      },
+      {
+        "title": "Started Web developing",
+        "date": "october 2021",
+        "icon": "globe",
+        "color": "blue",
+        "description": "got to learn about Web dev tools like Html, css, js, node and now playing with js/css frameworks"
```
Comment (Pradumnasaraf) [inline public/data/nathgoutam93.json:35]:
```suggestion
        "description": "Got to learn about Web dev tools like HTML, CSS, JS, node and now playing with JS/CSS frameworks"
```

File: public/data/nathgoutam93.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      {
+        "name": "Portfolio",
+        "url": "https://nathgoutam93.github.io/Portfolio/",
+        "icon": "link"
+      }
+    ],
+    "milestones": [
+      {
+        "title":"Started Game Developing",
```
Comment (Pradumnasaraf) [inline public/data/nathgoutam93.json:25]:
```suggestion
        "title":"Started Game Development",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: public/data/nathgoutam93.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        "url": "https://github.com/nathgoutam93",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/nathgoutam93",
+        "icon": "twitter"
+      },
+      {
+        "name": "Portfolio",
```
Comment (Panquesito7) [inline public/data/nathgoutam93.json:18]:
```suggestion
        "name": "Check out my portfolio",
```

**Comment by eddiejaoude** [comment]
@nathgoutam93 did you resolve the problem?

**Comment by nathgoutam93** [comment]
> @nathgoutam93 did you resolve the problem?

yes, It is resolved, it was an issue with the git configuration for my forked repo. the author was set to a different Github handler instead of mine

**Comment by eddiejaoude** [comment]
ok thank you for letting me know 👍

**Pull request closed without merging**